code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -698,7 +698,7 @@
 
 /datum/reagent/spraytan/proc/set_skin_color(mob/living/carbon/human/H)
 	if(H.dna.species.bodyflags & HAS_SKIN_TONE)
-		H.change_skin_tone(-30)
+		H.change_skin_tone(H.s_tone - 10)
```

Comment:
One possible issue with this is that there isn't actually a minimum, it just stops changing colour when it gets to `#000000`.
So if you start off with a skin tone of 220 and spray yourself twice, you get a shadowling:
![image](https://user-images.githubusercontent.com/57483089/133000193-b8fa6f97-d660-43d8-b85a-b43f9c1a4574.png)
Same issue with any other skin tone number, but it takes a bit more to get there.

I'd suggest changing it to have a minimum value of 220 or 230 to avoid this.
```suggestion
		H.change_skin_tone(max(H.s_tone - 10, -195))
```
<hr>

(Ingame `s_tone` is calculated as 35 minus whatever value you picked in character creation, so -195 is 230)
https://github.com/ParadiseSS13/Paradise/blob/c4fb435a8af82a60828a86e3e057db2a8b05d5ad/code/modules/client/preference/preferences.dm#L1738-L1742